i18n/de.toml

```diff
@@ -10,7 +10,7 @@ legendTopAutostart = "startet automatisch"
 legendTopName = "batterieunterstütztes Laden"
 legendTopSubline = "ohne Unterbrechungen"
 modalTitle = "Batterieeinstellungen"
-note = "Hinweis: Nur relevant im PV Modus. Das Ladeverhalten wird entsprechend angepasst."
+note = "Hinweis: Nur relevant im PV-Modus. Das Ladeverhalten wird entsprechend angepasst."
```

Comment:
Hier bin ich unsicher- gibts dafür eine orthografische Regel?

Reply:
@andig Ja, hier ist https://www.duden.de/sprachwissen/rechtschreibregeln/bindestrich#D28 anzuwenden.

Reply:
Dito, dann würde ich mich anschließen.

Reply:
Hab ich in Weblate angepasst.